pusher/src/Controller/AuthenticateController.ts

```diff
@@ -410,7 +411,7 @@ export class AuthenticateController extends BaseHttpController {
                     try {
                         const authTokenData: AuthTokenData = jwtTokenManager.verifyJWTToken(token as string, false);
                         if (authTokenData.accessToken == undefined) {
-                            throw Error("Token cannot to be check on Hydra");
+                            throw Error("Token cannot to be checked on Hydra");
```

Comment:
```suggestion
                            throw Error("Token cannot be checked on OpenID connect provider");
```

Reply:
The message seems wrong by the way.

We are checking if the JWT token contains the "accessToken" property.
So we are not making any request to the OpenID provider at this stage, right?

When can this happen? We should use "zod" to validate the structure of the "authTokenData"?
Right now, we are casting from "any" to "AuthTokenData", but this does not guarantee anything.